### DISCUSSION TITLE
[BUG] Latest version not released to Docker Hub

### DISCUSSION THREAD

**Discussion by RufusJWB** [Keyfactor/ejbca-ce#565] [General]
**Describe the Bug**

The latest release of EJBCA-CE wasn't released to [https://hub.docker.com/r/keyfactor/ejbca-ce/tags](Docker Hub). There 8.2.0.1 is the latest version which is missing some important bug fixes.

**To Reproduce**

Go to https://hub.docker.com/r/keyfactor/ejbca-ce/tags

**Expected Behavior**

Have EJBCA 8.2.1 available in Docker Hub.

**Screenshots and Logs**

![image](https://github.com/Keyfactor/ejbca-ce/assets/1633499/39e452ec-12dd-4f72-ab9a-ccb07e0cc1aa)

**Product Deployment** 

n/a

**Desktop**

n/a

**Additional Context**

n/a

**Comment by primetomas** [comment]
The CE releases does not necessarily follow the EJBCA Enterprise SW releases for minor releases.

**Reply by RufusJWB** [reply]
Do we have a chance of getting the latest CE version pushed to DockerHub soon? My problem is, that I'm using the CMP interface and the latest version at DockerHub has the bug, that the CMP configuration is disabled.

More generally speaking: do you have a release schedule for the container images of EJBCA-CE?

**Reply by primetomas** [reply]
I could not get a firm confirmation of dockerhub updates. The next feature release will be out in the end of may and will of course include fixes for everything, apart from new features such as hybrid certificates.
